cinema_service/urls.py

```diff
@@ -6,4 +8,4 @@
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("api/user/", include("user.urls", namespace="user")),
     path("__debug__/", include("debug_toolbar.urls")),
-]
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
The `static` function is used to serve media files during development. Ensure that in production, media files are served by a dedicated web server like Nginx or Apache.